modules/node-shared/src/main/resources/application.conf

```diff
@@ -100,3 +100,10 @@ fee-configs {
   }
 }
 
+
+addresses {
```

Comment:
Do we guarantee the consistency of such config entries in the cluster? I can imagine one node not setting these locked addresses in config so it will accept transactions from these addresses.

Afaik we do not validate locked addresses during consensus - we only do that on the endpoint when transactions are persisted in `TransactionStorage`. I think we should double-check if we reject transactions/blocks with addresses that are locked during consensus.

Reply:
There is a `private val addressesLockedAtOrdinal: Map[Address, SnapshotOrdinal] = Map.empty` in `BlockValidator` and I have a feeling that we should use addresses from config there as well instead of empty map 🤔

Reply:
good question, I see transactions are validated also in processProposal of BlockConsensusCell  and in BlockValidator (among other places) but I still need to understand the whole picture :)

Reply:
Yeah, I see that such a node's proposal will be considered as "Incorrect block" so I guess we should be fine. But worth testing on the cluster anyway :+1: